test.py

```diff
@@ -53,9 +53,6 @@ def gen_test_params():
             continue
         # Test encoding detection for each file we have of encoding for
         for file_name in listdir(path):
-            ext = splitext(file_name)[1].lower()
-            if ext not in ['.html', '.txt', '.xml', '.srt']:
```

Comment:
Ah, I made this change (opinionated admittedly) as this filter on suffix tripped me up and I couldn't work out why my tests weren't running.  It was unexpected behaviour to me.  It also appeared that it was functionally meaningless, as it does not actually exclude a single test from running, and so appeared to be a vestigial feature.  Happy to undo however, no biggie.